src/main/resources/db/changelog/changes/v2.1.0/cleanup.xml

```diff
@@ -76,9 +76,9 @@
     <changeSet id="MODPWD-64@@drop-set_validation_rules_md_json-function" author="psmahin">
         <sql dbms="postgresql">DROP FUNCTION IF EXISTS set_validation_rules_md_json</sql>
     </changeSet>
-    <changeSet id="MODPWD-64@@drop-f_unaccent-function" author="psmahin">
-        <sql dbms="postgresql">DROP FUNCTION IF EXISTS f_unaccent</sql>
+    <changeSet id="MODPWD-64@@drop-f_unaccent-function" author="psmahin" runOnChange="true">
+        <sql dbms="postgresql">SELECT * FROM information_schema.routines WHERE routine_name LIKE '%f_unaccent'</sql>
```

Comment:
could you please explain what you're trying to achieve with SELECT?
as I understand the outcome of this will be NOTHING which is weird
what will be the result if a schema DOES have `f_unaccent` function? I assume it won't be deleted, right?